irods/helpers/__init__.py

```diff
@@ -56,3 +58,21 @@ def temporarily_assign_attribute(
             setattr(target, attr, save)
         else:
             delattr(target, attr)
+
+
+def get_data_object(sess, logical_path):
+    try:
+        # Check for a data object at the normalized path.
+        return sess.data_objects.get(iRODSPath(logical_path))
+    except ex.DataObjectDoesNotExist:
+        return None
+
+
+def get_collection(sess, logical_path):
+    try:
+        # Path normalization is internal to this call.
+        return sess.collections.get(logical_path)
+    except ex.CollectionDoesNotExist:
+        return None
+    except Exception as e:
+        return e
```

Comment:
Are these part of the public API or strictly for testing?

Reply:
Okay. Please add a docstring to each function so users understand how they are meant to be used.

Reply:
will do.